README.md

```diff
@@ -36,29 +36,25 @@ Profile based segregation
 
 We are using maven build profiles to segregate the huge list of individual projects we have in our repository.
 
-As for now, vast majority of the modules require JDK8 to build and run correctly.
-
-The projects are broadly divided into 3 lists: first, second and heavy. 
+The projects are broadly divided into 3 lists: default, default-jdk8 and default-heavy. 
 
 Next, they are segregated further on the basis of the tests that we want to execute.
 
 Additionally, there are 2 profiles dedicated for JDK9 and above builds - **which require JDK 17**.
 
 We also have a parents profile to build only parent modules.
 
-Therefore, we have a total of 9 profiles:
+Therefore, we have a total of 7 profiles:
 
-| Profile                    | Includes                    | Type of test enabled |
-| -------------------------- | --------------------------- | -------------------- |
-| default-first              | First set of projects       | *UnitTest            |
-| integration-lite-first     | First set of projects       | *IntegrationTest     |
-| default-second             | Second set of projects      | *UnitTest            |
-| integration-lite-second    | Second set of projects      | *IntegrationTest     |
-| default-heavy              | Heavy/long running projects | *UnitTest            |
-| integration-heavy          | Heavy/long running projects | *IntegrationTest     |
-| default-jdk9-and-above     | JDK9 and above projects     | *UnitTest            |
-| integration-jdk9-and-above | JDK9 and above projects     | *IntegrationTest     |
-| parents                    | Set of parent modules       | None                 |
+| Profile           | Includes                    | Type of test enabled |
+|-------------------|-----------------------------| -------------------- |
+| default           | JDK9 and above projects     | *UnitTest            |
```

Comment:
Let's reword this to JDK17 and above projects

Reply:
ok